helm/managed/logging/loki-stack/README.md

```diff
@@ -0,0 +1,98 @@
+## This folder contains lightweight Loki Stack deployment chart and HPCC Systems preferred values
+
+<table>
```

Comment:
Is this relevant?  I think left in by mistake.

Reply:
> Is this relevant? I think left in by mistake.

I should have added a comment, it's meant to be a placeholder for alerts/messages for our users akin to this message:
https://github.com/hpcc-systems/HPCC-Platform/tree/master/helm/managed/logging/elastic#this-folder-contains-lightweight-elastic-stack-deployment-chart-and-hpcc-systems-preferred-values